app/utils/captcha.js

```diff
@@ -47,14 +47,20 @@ async function verifyCAPTCHAEnterpriseVersion (token) {
     const score = riskAnalysis.score
     const reasons = riskAnalysis.reasons || []
 
-    if (typeof score !== 'number' || score < 0.9) {
+    if (typeof score !== 'number' || score < 0.5) {
       logger.info('Failed reCAPTCHA response', {
         tokenProperties: body.tokenProperties || {},
         score: score || 'N/A',
         reasons
       })
       return false
     }
+
+    logger.info('Successful reCAPTCHA response', {
+      tokenProperties: body.tokenProperties || {},
+      score: score || 'N/A',
```

Comment:
Presumably it’s impossible for `score` to be undefined if we get here.

Reply:
If it is undefined - then it will just print 'N/A'